src/config.ts

```diff
@@ -177,6 +177,11 @@ const DEFAULTS = o({
         h: "help",
         authors: "credits",
         openwith: "hint -W",
+        styleinclude: "style -i",
+        styledelete: "style -x",
+        styledisable: "style -d",
+        styleenable: "style -e",
+        styletoggle: "style -t",
```

Comment:
Could we try to match the Pentadactyl (and I presume Vimperator) commands?

```
:sty[le] [-name={name}] [-group={name}] [-agent] {filter} {css}:sty[le] -name={name} -append {filter} {css}:sty[le] [-name={name}] [-group={name}] [filter]
        
            Add CSS styles to the browser or to web pages. {filter} is a
            comma-separated list of site-filters for which the style will
            apply. The ! character may not be used to invert the sense
            of the match. [css] is a full CSS rule set (e.g., body {
            color: blue; }).
        

        The following options are available:
        -appendIf provided along with -name, {css} and
                {filter} are appended to its current value. (short name
                -a)-agentIf provided, the style is installed as an Agent sheet, which
                applies to contents user interface widgets as well as normal
                elements. (short name -A)-group={group}The group to which to add this style. Please note that
                this grouping is for semantic and cleanup purposes only. No
                additional site filtering is applied.-name={name}If provided, any existing style with the same name is
                overridden, and the style may later be deleted using
                {name}. (short name -n)

        If [css] isn't provided, matching styles are listed.
    

:dels
:delstyle
:dels[tyle][!] [-name={name}] [-index={index}] [filter] [css]
        
            Delete any matching styles. With [!], delete all styles.
            If [filter] is provided, only matching elements of the
            filter are disabled. For instance, when run with a filter
            mozilla.org, an existing style with a filter
            www.google.com,mozilla.org, will result in a style for
            www.google.com.
        

        The available options are:

        -nameThe name provided to :style (short name -n)-indexFor unnamed styles, the index listed by :style
                (short name -i)
    

:styleenable
:stylee
:styenable
:stye
:stylee[nable][!] [-name={name}] [-index={index}] [filter] [css]
        Enable any matching styles. With [!], enable all styles. Arguments are the same as for :delstyle
    

:styledisable
:styled
:stydisable
:styd
:styled[isable][!] [-name={name}] [-index={index}] [filter] [css]
        Disable any matching styles. With [!], disable all styles. Arguments are the same as for :delstyle
    

:styletoggle
:stylet
:stytoggle
:styt:
stylet[oggle][!] [-name={name}] [-index={index}] [filter] [css]
        Toggle any matching styles. With [!], toggle all styles. Arguments are the same as for :delstyle
```

Thanks.

Reply:
I'll try to do initial css load on autocmd as well...